pkg/slack/client.go

```diff
@@ -256,5 +278,14 @@ func (c *Client) doRequest(ctx context.Context, url string, res interface{}, met
 		return err
 	}
 
+	if !res.IsOk() && strings.Contains(res.ErrorText(), "ratelimited") {
```

Comment:
Shouldn't we check for http status code 429 instead of looking in the error text?

Reply:
I'm game